sonarqube-sample-java/src/main/java/com/consoleconnect/mw/sonarqube/sample/entity/User.java

```diff
@@ -4,6 +4,8 @@
 
 public class User {
 
+  private int b;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "b" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sq.ci.consoleconnect.com/coding_rules#rule_key=squid%3AS1068)